typescript/src/scss/main.scss

```diff
@@ -1,16 +1,84 @@
+
+  $light_blue: rgb(136, 206, 222);
+  $medium_gray: rgb(237, 237, 237);
+  $dark_gray: rgb(96, 96, 96);
+  $white: rgb(255, 255, 255);
+  $low_opacity_gray: rgb(96, 96, 96, 0.1);
+
 .metalation_calculator {
   #metalation_table {
     table-layout: fixed;
     width: 100%;
-    border-collapse: collapse;
+    border-collapse: separate;
     margin: 20px 0;
+    border-spacing: 0;
 
     td, th {
-      text-align: right;
+      text-align: center;
       padding: 2px 5px;
-      vertical-align: top;
       border: 0;
+      justify-content: center;
+    }
+    thead, tfoot {
+      background-color: $light_blue;
+      padding: 1.2%
+    }
+    td, .result {
+      text-align: right;
+      padding-right: 20px;
     }
+    tr:nth-child(even) {
+      background-color: $medium_gray;
+    }
+
+    .grouped {
+      background-color: $low_opacity_gray;
+    }
+
+    /* Tooltip text */
+    .tooltip{
+      position: relative;
+    }
+
+    .tooltip .tooltiptext {
+      visibility: hidden;
+      width: 120px;
+      background-color: $dark_gray;
+      color: $white;
+      text-align: center;
+      padding: 5px 0;
+      border-radius: 6px;
+      position: absolute;
+      z-index: 1;
+      display: block;
+      bottom: 100%;
+    }
+
+    /* Show the tooltip text when you mouse over the tooltip container */
+    .tooltip:hover .tooltiptext {
+      visibility: visible;
+    }
+    .tooltip .tooltiptext::after {
+      content: " ";
+      position: absolute;
+      top: 100%; /* At the bottom of the tooltip */
+      left: 50%;
+      margin-left: -5px;
+      border-width: 5px;
+      border-style: solid;
+      border-color: $dark_gray transparent transparent transparent;
+    }
+
+    .right_spacing_odd {
+      border-right: 15px solid $white;
+    }
+    .right_spacing_even{
+      border-right: 15px solid $medium_gray;
+    }
```

Comment:
So what I meant in the other comment is that you could have `.right_spacing` classes inside `tr` and `tr:nth-child(even)` instead of needing separate classes.

Reply:
<img width="380" alt="Screenshot 2021-11-23 at 09 37 06" src="https://user-images.githubusercontent.com/73976500/143001601-94fb26d3-cd55-465c-8739-f2b8f3ae257b.png">
This doesn't work if this is what you meant.

Reply:
What you've got is selecting `<tr>`s with the `right_spacing` class, not the `<td>`s inside them.

How about:

```scss
tr:nth-child(even) {
   ...(what's currently in there)
   
  td.right_spacing {
    border-right: 15px solid $white;
  }
}
```